From 7eb8bd8f01d9e5d8903be19a6dcc0aa903760989 Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@gmail.com>
Date: Wed, 29 Jun 2016 14:27:44 +0800
Subject: [PATCH 1113/5242] MLK-12931 lcdif: fix "lcd-3v3: Failed to request
 enable GPIO91: -16"

commit  bf9c213086156317b7ac4b3ce42e9bba27300472 from
https://source.codeaurora.org/external/imx/linux-imx.git

The PIN is conflicted with CSI camera. Need to disable CSI to make lcdif work.

Signed-off-by: Robby Cai <robby.cai@gmail.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
index dd8c1da..9391e75 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
@@ -33,3 +33,7 @@
 		status = "okay";
 	};
 };
+
+&ov5640 {
+	status = "disabled";
+};
-- 
1.7.9.5

